plugins/modules/pacemaker_resource.py

```diff
@@ -27,13 +27,14 @@
   state:
     description:
       - Indicate desired state for cluster resource.
-    choices: [present, absent, enabled, disabled]
-    default: present
+      - The state V(cleanup) has been added in community.general 12.0.0.
```

Comment:
New features are added in 11.x.0 minor releases, so it would be:
```suggestion
      - The state V(cleanup) has been added in community.general 11.2.0.
```